### PR TITLE
Update VEP config

### PR DESCRIPTION
**Pull request by veidenberg**
### Description
Update VEP config to support unsorted input and use the new workflow repo (+ cleanup of unused nextflow payload params). 

### Related JIRA Issue(s)
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-2709

### CODE REVIEW

File: app/vep/models/pipeline_model.py [hunk 1]
```diff
@@ -55,11 +48,13 @@ class ConfigIniParams(BaseModel):
     biotype: bool = False
     transcript_version: int = 1
     canonical: int = 1
-    gff: str = "/nfs/public/rw/enswbsites/dev/vep/test/genes.gff3.bgz"
+    sort: int = 1
+    # TODO: use genome_id from client payload to set gff+fasta paths
```
Comment (Mehrnaz-Charkhchi) [inline app/vep/models/pipeline_model.py]: We have already discussed these hardcoded parts, Are these comments necessary? is there an actual update here that I'm missing?

Comment (veidenberg) [reply app/vep/models/pipeline_model.py]: Just a reminder that dynamic paths need to be implemented ~~for beta-2 (i.e. in this sprint?)~~. The comments are not the update but the changes in VEP config in order to work with the new pipeline (goes with the other PRs; see the PR description and linked JIRA ticket).

Comment (azangru) [reply app/vep/models/pipeline_model.py]: > dynamic paths need to be implemented for beta-2

Erm, isn't beta2 happening earlier/independently of VEP?

> (i.e. in this sprint?)

We would need loaded production data, an up-to-date grpc service probably pointing at the updated database, plus updates to the web-metadata-api that will pick the new attributes from the grpc response. Sounds unlikely for this sprint.

Comment (veidenberg) [reply app/vep/models/pipeline_model.py]: Yes, you're right, sorry for the confusion. No need to worry about it now. Comments removed.

**Pull request closed without merging**